supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -212,26 +209,52 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     if (referenceName.length <= 127) referenceName else referenceName.md5Sum
   }
 
+  private[jes] def findGlobOutputs(jobDescriptor: BackendJobDescriptor): Seq[WdlGlobFile] = {
```

Comment:
I find it amusing that you're against extra type safety because you hate the extra keystrokes of `foo.x` instead of `x` but you love typing `traversable` everywhere ;)